### PR TITLE
Updated api strings to make things work

### PR DESCRIPTION
**Pull request by yinyinnnnn**

### CODE REVIEW

**Comment by lucidDreamss** [comment]
thank you so much <3

**Comment by zerotypic** [comment]
Thanks @yinyinmeimei! I'm still unable to test it out at the moment, @lucidDreamss I assume it fixes the issue you were having? Will merge if I get confirmation that it is working now.

**Comment by yeejian-tan** [comment]
Just to add on, this worked fine for me, although with a few warnings about the api version, I presume:
```
Invalid server response: {<otp_retrieval_response>}
Warning: Unexpected server response, key 'version' is '2.0', not '1.7'.
Got success code: <success_code>
OTP will be sent to mobile phone number <phone_no>
Enter OTP to continue: <otp>
Invalid server response: {<response_body>}
Warning: Unexpected server response, key 'version' is '2.6', not '2.2'.
Successfully decrypted using date 20211129.
Decryption key: <blah>
Nonce: <blah>
userid tag: <blah>
password tag: <blah>
Credentials:
<creds>
```

File: wasg-register.py [hunk 1]
```diff
@@ -19,12 +19,12 @@
 # Test URL is for debugging.
 ISP_CONFIG = {
     "singtel" : {
-        "essa_url" : "https://singtel-wsg.singtel.com/essa_r11",
+        "essa_url" : "https://singtel-wsg.singtel.com/essa_r12",
         "create_api_versions" : ("2.4", "2.4"),
         "retrieve_api_versions" : ("1.7", "2.2")
```
Comment (yeejian-tan) [inline wasg-register.py]:
```suggestion
        "retrieve_api_versions" : ("2.0", "2.6")
```

Comment (yeejian-tan) [reply wasg-register.py]: This removes the errors.

**Comment by zerotypic** [comment]
Ok merging this. Thanks @yinyinmeimei and @SwampertX!

**Pull request merged**